apps/nextjs-website/src/components/FramedVideo.tsx

```diff
@@ -0,0 +1,34 @@
+import { FramedVideo as FramedVideoRC } from '@react-components/components';
+import { FramedVideoProps } from '@react-components/types';
+import { FramedVideoSection } from '@/lib/fetch/types/PageSection';
+
+const makeFramedVideoProps = ({
+  videoURL,
+  video,
+  text,
+  sectionID,
+  theme,
+}: FramedVideoSection): FramedVideoProps => ({
+  sectionID,
+  theme,
+  videoURL: videoURL ?? '',
+  video: video
+    ? {
+        srcURL: video.srcURL ?? '',
+        autoplay: video.autoplay,
+        loop: video.loop,
+        showControls: video.showControls,
+        fallback: video.fallback,
+        playButtonLabel: video.playButtonLabel,
+        pausedPlayButtonLabel: video.pausedPlayButtonLabel,
+      }
+    : undefined,
```

Comment:
```suggestion
  ...(video.data && { videoURL: video.data.attributes.url },
```
Figma does not show any need for overlay text or anything of the sort.
Simply pass the videoURL (overwriting videoURL to give video priority).